DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/ExceptionUsage.scala

```diff
@@ -52,24 +52,22 @@ object ExceptionUsage extends ProjectAnalysisApplication {
             //    of all exceptions (objects that are an instance of Throwable)
             //    (Since exceptions are never created in a register, it is sufficient
             //    to analyze the operands.)
-            def collectExceptions(value: result.domain.DomainSingleOriginReferenceValue): Unit = {
+            def collectExceptions(value: result.domain.DomainSingleOriginReferenceValue): Unit =
```

Comment:
I'm all for removing unnecessary brackets, but I think curlies around multi-line statements help with code comprehensions. Is it possible to keep them?

Reply:
[https://scalameta.org/scalafmt/docs/configuration.html#redundantbraces-defnbodies](This Rule and the next one I think)